app/javascript/tagging/tests/tagging.test.js

```diff
@@ -100,4 +104,44 @@ describe('Tagging component without redux mapping', () => {
     wrapper.instance().onTagDeleteClick('wowo');
     expect(onTagCategoryChange.mock.calls).toHaveLength(1);
   });
+
+  it('should call methods is multi', () => {
+    const onTagCategoryChange = jest.fn();
+    const onTagValueChange = jest.fn();
+    const onSingleTagValueChange = jest.fn();
+    const onTagDeleteClick = jest.fn();
+    const onTagMultiValueChange = jest.fn();
+    const wrapper = shallow(<Tagging
+      tags={tags}
+      assignedTags={assignedTags}
+      onTagValueChange={onTagValueChange}
+      onSingleTagValueChange={onSingleTagValueChange}
+      onTagMultiValueChange={onTagMultiValueChange}
+      onTagCategoryChange={onTagCategoryChange}
+      onTagDeleteClick={onTagDeleteClick}
+      selectedTagCategory={selectedTagCategory1}
+      selectedTagValue={selectedTagValue}
+    />);
+    expect(wrapper.find('ValueModifier').props().multiValue).toBe(true);
+  });
+
+  it('should call methods11', () => {
```

Comment:
Is this a typo or is `methods11` an actual method?

Reply:
typ, fixed it.